agreement_legal/migrations/12.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(env, version):
```

Comment:
```suggestion
@openupgrade.migrate()
def migrate(env, version):
```

Reply:
please do not add unecessary dependencies.

Reply:
Yes. I checked with @max3903 and It is not necessary.